### ISSUE TITLE
NVDA doesn't appear to support HTML figure element or ARIA figure role

### ISSUE THREAD

**Issue by scottaohara** [nvaccess/nvda#9177]
### Steps to reproduce:
With default NVDA settings, go to: [this `figure` test page](https://scottaohara.github.io/testing/figure/tests.html).

Navigate in browse mode to any of the tests. (easiest way is to navigate by heading and then navigate with the down arrow).

Other screen readers like VoiceOver on macOS and JAWS with Chrome and Firefox (and IE11 if `role=figure` is used) will announce the `figure` role and its accessible name (from the `figcaption` or an `aria-label`).

### Actual behavior:
NVDA does not announce the figure role, nor does it indicate an association between a `figure`'s content and its associated caption.

### Expected behavior:
I would expect the native HTML `<figure>`, or ARIA's figure role to be announced in some way, helping to convey the relationship between the figure's primary content and its caption.

#### NVDA installed/portable/running from source:
installed

#### NVDA version:
2018.4.1

#### Windows version:
Windows 10 Pro
10.0.17134

#### Name and version of other software in use when reproducing the issue:
Firefox 64.0.2, Chrome 71, IE11, Edge 42

#### Other information about your system:
...

#### Does the issue still occur after restarting your PC?
yes

#### Have you tried any other versions of NVDA?
No
